tests/cupyx_tests/scipy_tests/ndimage_tests/test_measurements.py

```diff
@@ -11,10 +11,17 @@
 import cupyx.scipy.ndimage  # NOQA
 
 try:
+    import scipy
     import scipy.ndimage  # NOQA
+    scipy_version = numpy.lib.NumpyVersion(scipy.__version__)
 except ImportError:
     pass
```

Comment:
Maybe we need this? I am sure there's a less hackier solution 😅
```suggestion
    scipy_version = numpy.lib.NumpyVersion('0')
```

Reply:
Thanks. I have added it, but with '0.0.0' since '0' is not considered a valid version string by `NumpyVersion`. I used that function because it evaluates things like `numpy.lib.NumpyVersion('1.5.0') > '1.5.0rc1'` to `True` while regular string comparison would not.  Another option is to use `packing.packing.Version` which can also handle cases like this.

I thought adding a list of function names at the top would be better than adding a version check and `pytest.skip` to each individual test method in the `TestStats` class.

Reply:
> I thought adding a list of function names at the top would be better than adding a version check and `pytest.skip` to each individual test method in the `TestStats` class.

This makes sense because the `TestStats` class does not subclass `unittest.TestCase` so we can't use `setUp()` to do this for all test methods.